packages/account/src/Components/Routes/__tests__/binary-link.spec.tsx

```diff
@@ -49,13 +44,9 @@ describe('<BinaryLink />', () => {
 
         expect(() =>
             render(
```

Comment:
**NIT:** This render could be written as a function and then re-used in the test suites